web/src/components/icons/sponsors/Cloudflare.tsx

```
@@ -0,0 +1,24 @@
+import type { SVGProps } from 'react'
+
+export default function Cloudflare(props: SVGProps<SVGSVGElement>) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 101.4 33.5" {...props}>
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)